Frameworks/UIKit/UIPickerView.mm

```diff
@@ -357,7 +363,7 @@ - (int)_selectedRow {
 }
 
 - (id)_cellSelectedUp:(id)cell {
```

Comment:
in UITableViewInternal.h, it defines 
- (void)_cellSelectedUp:(UITableViewCell*)cell;